plugin/evm/order_api.go

```diff
@@ -0,0 +1,48 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"context"
+	"encoding/hex"
+	"strings"
+
+	"github.com/ava-labs/subnet-evm/plugin/evm/orderbook"
+	hu "github.com/ava-labs/subnet-evm/plugin/evm/orderbook/hubbleutils"
+)
+
+type OrderAPI struct {
+	tradingAPI orderbook.TradingAPI
+	vm         *VM
+}
+
+func NewOrderAPI(tradingAPI orderbook.TradingAPI, vm *VM) *OrderAPI {
+	return &OrderAPI{
+		tradingAPI: tradingAPI,
+		vm:         vm,
+	}
+}
+
+type PlaceOrderResponse struct {
+	Success bool `json:"success"`
+}
+
+func (api *OrderAPI) PlaceSignedOrder(ctx context.Context, rawOrder string) (PlaceOrderResponse, error) {
```

Comment:
Is this being used?

Reply:
were you able to plug this in the RPC interface which mms will call to place orders?

Reply:
nice.